lib/tasks/clean_up_orphaned_fixity_records.rake

```diff
@@ -0,0 +1,56 @@
+# Context:
+# When a fileset is deleted from Fedora, the records for its fixity checks remain in the database.
+# This creates confusion when attempting to ensure that all current files have had their fixity
+# checked recently.
+#
+# What this job does:
+# Loop through all fixity records and verify that they are associated with a fileset that we
+# still have on hand. If it turns out they are not, delete the "orphaned" fixity check record.
+
+namespace :chf do
+  desc "run fixity checks with logs and notification on failure"
+  task :clean_up_orphaned_fixity_checks => :environment do
+    @conn = Blacklight.default_index.connection
+    ChecksumAuditLog.find_in_batches(batch_size: 100 ) do | batch |
```

Comment:
Thanks for the helpful advice. Refactored to use find_in_batches. Gee, SOLR is really really fast.